ophyd/ophydobj.py

```diff
@@ -60,6 +58,13 @@ def __init__(self, *, name=None, parent=None):
         self._args_cache = {k: None for k in self.subscriptions}
         # count of subscriptions we have handed out, used to give unique ids
         self._cb_count = count()
+        # Create logger name from parent or from module class
+        if self.parent:
+            base_log = self.parent.log.name
+        else:
+            base_log = self.__class__.__module__
+        # Instantiate logger
+        self.log = logging.getLogger(base_log + '.' + self.name)
```

Comment:
Won't this lead to geometrically growing set of names as the child name is `'{parent.name}_{attr}'`?

Reply:
:sheep: never mind, missed the `.log.` in the line where we set `base_log`.